### ISSUE TITLE
[feature request] Add fc-selectable class

### ISSUE THREAD

**Issue by frlinw** [fullcalendar/fullcalendar#4630]
## Feature Requests
Currently you have to watch the selectable option to toggle a custom class.
A native 'fc-selectable' class toggled on the fullcalendar root node could be nice.

fullcalendar already do it with fc-unselectable.
Just a question about it, why is it set on the <body> tag instead of the fullcalendar root node ?
![image](https://user-images.githubusercontent.com/11896237/55885711-b756d580-5baa-11e9-93a4-075b758f8b2c.png)

**Comment by acerix** [comment]
Are you looking for a callback when the `selectable` option is changed?

What you're describing doesn't sound related to `fc-unselectable` which makes it so text in the calendar [can't be selected](https://developer.mozilla.org/en-US/docs/Web/CSS/user-select).

**Comment by frlinw** [comment]
Not at all, as I said, just a class toggled on the fullcalendar root node when the option is active (even if there is no style attached to it natively).
I know the purpose of fc-unselectable, I talked about it because it's related to selectable option and it's the same "kind of" things (toggle a class).

I take an old example where I did it manually: https://codepen.io/frlinw/pen/zXzqwR

**Comment by acerix** [comment]
Ok, so you are looking to be able to change the CSS in the calendar depending on the value of the `selectable` option?  It seems to me that is better done as in your example instead of building that into the calendar.

**Comment by frlinw** [comment]
Indeed, It will give the opportunity to add styles depending on a fullcalendar's state without requiring a custom implementation.
It's useful if, for example, you want to share a custom theme for fullcalendar.

**Comment by EinarValholl** [comment]
@acerix are you aware that `.fc-unselectable` class and preventing `selectstart` events via `el.addEventListener('selectstart', preventDefault); ` prevents user from any interaction on custom components generated instead of default one? E.g. in case of event template has some form it's not possible to focus on those fields?
